tests/test_plotting.py

```diff
@@ -569,6 +569,45 @@ def test_vsel_iterable(mapdl, make_block):
     )
 
 
+def test_color_areas(mapdl, make_block):
+    pl = mapdl.aplot(vtk=True, color_areas=True, return_plotter=True)
+
+
+# This is to remind us that the pl.mesh does not return data for all meshes in CICD.
+@pytest.mark.xfail
+def test_color_areas_fail(mapdl, make_block):
+    pl = mapdl.aplot(vtk=True, color_areas=True, return_plotter=True)
+    assert len(np.unique(pl.mesh.cell_data["Data"], axis=0)) == mapdl.geometry.n_area
```

Comment:
@AlejandroFernandezLuces 

for some reason, locally this test works... 
When we query ``pl.mesh.cell_data["Data"]`` we obtain an array with the colours. When we only get the uniques (``np.unique``) it should match the amount of areas, because PyMAPDL choose a different color per area (up to 255 colors) if you use ``color_areas=True``.

But on the CICD, this only returns an array with all the same values:

https://github.com/ansys/pymapdl/actions/runs/5644861964/job/15289519572#step:12:8270

I have the feeling that ``xvfb`` library is avoiding the plot to a level that no colors are ever assigned to the cell? I'm not sure.

Reply:
It's throwing some errors in my local too: 
![image](https://github.com/ansys/pymapdl/assets/17165476/d53a9d17-c645-44da-9bd9-4f4afd2e5f44)

I'll take a deeper look tomorrow, there might be incompatibilities depending on the OS?

Reply:
I'm using MacOS.... with MAPDL ubuntu image, but effectively, pyvista is only seeing the MacOS.

Reply:
I didn't have the proper PyMAPDL version when running in my local. It does work fine on my Windows too. I see you are already skipping the tests. 

It might be worth a try to use this pytest plugin I just found: https://github.com/The-Compiler/pytest-xvfb It is configurable, just in case the issue is with some `xvfb` color configuration that is missing. Also it might throw (better) errors when `xvfb` is not behaving as expected.

Reply:
why always me man.... :( 

I'll just go ahead and merge since everything pass.